### PR TITLE
Fix stability issue in install flow and add support for live OCS version

### PR DESCRIPTION
**Pull request by shyRozen**

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts [hunk 1]
```diff
@@ -25,8 +25,14 @@ const status = statuses.get(0);
 // Create storage cluster page
 const btnCreate = $('.pf-m-primary');
 const selectAllBtn = $('[aria-label="Select all rows"]');
+const live = process.env.OCS_LIVE;
+let CATALOG_SRC;
+if (live === '1') {
+  CATALOG_SRC = 'redhat-operators';
+} else {
+  CATALOG_SRC = 'ocs-catalogsource';
+}
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts]: nit: 
```
let CATALOG_SRC = 'ocs-catalogsource`;
if (live) CATALOG_SRC = 'redhat-operators';
```

Comment (shyRozen) [reply frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts]: Thanks! Changed.

**Pull request merged**